components/mdx/Spoiler.test.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
```

Comment:
Just to keep the same standard between all React code.

```suggestion
import React from 'react'
```